src/app.js

```diff
@@ -276,11 +276,16 @@ export function app(app) {
           removeElement(element, reusableChild[0], reusableNode)
         }
       }
-    } else if (node !== oldNode) {
+    } else if (
+      element != null &&
+      node !== oldNode &&
+      node !== element.nodeValue
+    ) {
       var i = element
       parent.replaceChild((element = createElement(node)), i)
```

Comment:
How, I am looking around patch to find the conversion ^^'
I am partially discovering hyperapp internal with questions. So, sometimes I miss things when reading at code that raise questions later.